### PR TITLE
Rename everything to the new name

### PR DESCRIPTION
**Pull request by thockin**
We'll rename the repo as a last step.

### CODE REVIEW

**Comment by vmarmol** [comment]
LGTM, we'll need to start working on the new logo.

File: Vagrantfile [hunk 1]
```diff
@@ -7,12 +7,12 @@ VAGRANTFILE_API_VERSION = "2"
...
 Vagrant.require_version ">= 1.6.2"
 
-if ARGV.first == "up" && ENV['USING_KUBE_SCRIPTS'] != 'true'
+if ARGV.first == "up" && ENV['USING_LMKTFY_SCRIPTS'] != 'true'
   raise Vagrant::Errors::VagrantError.new, <<END
 Calling 'vagrant up' directly is not supported.  Instead, please run the following:
 
-  export KUBERNETES_PROVIDER=vagrant
-  ./cluster/kube-up.sh
+  export LMKTFYRNETES_PROVIDER=vagrant
```
Comment (erictune) [inline Vagrantfile]: bad substitution

**Comment by erickhan** [comment]
LMKTFY, LGTM!

**Comment by erictune** [comment]
One place where search-and-replace did the wrong thing.  Other than that, LGTM.

File: Vagrantfile (hunk 1 above)
Comment (thockin) [inline Vagrantfile]: fixed

**Comment by sub-mod** [comment]
@thockin great +100 , LGTM .

**Comment by zmerlynn** [comment]
Needs a rebase.

**Comment by thockin** [comment]
The rebase is going to be a nightmare, so collecting feedback first.

**Comment by zmerlynn** [comment]
We should add a suggested pronunciation, too. Maybe you could record something, like Linus did?

**Comment by derekwaynecarr** [comment]
Can't wait to type cluster/lmktfy-up !

**Comment by justinsb** [comment]
LGTM.  You should probably register lmktfy.eu before someone grabs it.

**Comment by thockin** [comment]
re: pronunciation.  "lem-kut-fee" is totally different than "lem-cut-fee" and anyone who can't hear it immediately should have their ears checked.

**Comment by vishh** [comment]
:100:

**Comment by pweil-** [comment]
:trophy: :trollface:

**Comment by markturansky** [comment]
Now to change all my dotfiles and shell commands.

@pweil- will you include this in the OS rebase?

**Comment by liggitt** [comment]
For backwards compatibility, we should continue honoring .kubeconfig files. That would be a simple 8-step merging load:

- `--lmktfyconfig`
- `--kubeconfig`
- $LMKTFYCONFIG
- $KUBECONFIG
- $CWD/.lmktfyconfig
- $CWD/.kubeconfig
- ~/.kube/.kubeconfig
- ~/.lmktfy/.lmktfyconfig

**Comment by baldwinSPC** [comment]
+1 ;)

**Comment by dchen1107** [comment]
LGTM on everything except the logo. Can we hire a graph designer for the new logo?

**Comment by thockin** [comment]
Based on the overwhelming support for this idea, and the fact that I am in a bad mood, I am killing this proposal.

**Comment by bprashanth** [comment]
2 hours to merge window. rebase already.

**Comment by vmarmol** [comment]
I think @thockin took offense to @dchen1107's comment about getting a "real" graphic designer. The logo is great @thockin I promise! No one thinks it looks like a box of tissues I swear.

**Comment by erickhan** [comment]
well, the poor logo is my mea culpa!

anyway, let-me-logo-that-for-you will be the next project.

**Pull request closed without merging**